### ISSUE TITLE
Failures of current large-scale ABM approaches...

### ISSUE THREAD

**Issue by davidrpugh** [ScalABM/requirements#2]
I think we need to draw a sharper contrast between ScalABM and existing large-scale ABM modeling efforts (in particular the various flavors of EURACE, the CRISIS project, JMAB, etc). Various flavors of the EURACE project use the [FLAME](http://www.flame.ac.uk/) platform which is scalable but requires models written in C or C++, has an extremely high learning curve, is not widely used outside of academia, and is currently not being actively developed.  I am not sure what the EURACE projects are doing on the estimation, calibration, validation front.  CRISIS project and JMAB are not single-thread models and therefore not inherently scalable.  Neither project has done much on the calibration, estimation, and validation front (at least to my knowledge).

While we want to contrast our approach with those existing projects, we do not want to be too critical.  Ultimately, it would be great if we could bring those groups into the fold.

**Comment by prauwolf** [comment]
I agree we should draw a sharper contrast between existing models, however, I need to spend some more time researching the above models before I feel comfortable juxtaposing them to ScalABM. I'll update the document after spending a bit more time perusing the existing ABM projects.
